BankSDK/GiniBankSDK/Tests/GiniBankSDKTests/GiniBankSDKTests.swift

```diff
@@ -10,6 +10,44 @@ import XCTest
 @testable import GiniBankSDK
 
 final class GiniBankSDKTests: XCTestCase {
```

Comment:
The test class name is too generic. You could rename it to `String+UtilsTests` for example.